src/Pillar-ExporterCore.package/PRDocumentListTransformer.class/instance/sort.by..st

```diff
@@ -0,0 +1,5 @@
+sorting
+sort: aCollectionOfReferences by: aString
+	aString = 'date' ifTrue: [ ^ PRDocumentListDateSorter with: aCollectionOfReferences ].
+	aString = 'name' ifTrue: [ ^ PRDocumentListNameSorter with: aCollectionOfReferences ].
```

Comment:
We will have to see how to sort by default, and what happens if there is an invalid value.